src/_client.ts

```diff
@@ -76,6 +76,13 @@ export interface ChalkClientOpts {
    */
   fetchHeaders?: typeof Headers;
 
+  /**
+   * The format to use for date-type data.
+   *
+   * Defaults to "ISO_8601" (in UTC), also supports "EPOCH_MILLIS"
```

Comment:
Clarification question - this is changing the default deserialization format, i.e. once a customer updates to this version of the chalk client, they will need to update their client instantiation with `EPOCH_MILLIS` or any existing workflow they had that depended on the existing timestamp format will break, right? I think strictly speaking this would make this a breaking change.

Reply:
`ISO_8601` is what we return in most cases (all single query time formats, dates for bulk queries) and `EPOCH_MILLIS` is only returned for timestamps in bulk/multi queries

This is a breaking change 😞 but the breaking change is also the bug. I'm not sure whether this gets a patch for being a bugfix, a minor version for new functionality to mitigate the breaking change, or a major version for changing the default behavior of the SDK